tcms/telemetry/static/telemetry/js/testing/breakdown.js

```diff
@@ -0,0 +1,122 @@
+$(document).ready(() => {
+    $('.selectpicker').selectpicker();
+
+    loadInitialProduct();
+
+    document.getElementById('id_select_product').onchange = updateTestPlanSelect;
+    document.getElementById('id_select_test_plan').onchange = reloadCharts;
+});
+
+function loadInitialProduct() {
+    jsonRPC('Product.filter', {}, data => {
+        updateSelect(data, '#id_select_product', 'id', 'name');
+        reloadCharts();
+    });
+}
+
+function updateTestPlanSelect() {
+    const productId = $('#id_select_product').val();
+
+    if (!productId) {
+        reloadCharts();
```

Comment:
Clear the list of selected test plans before reloading charts. To reproduce:

1) Select 1 product and see what TPs are available
2) Select ---

Result: Test plan select is not cleared.